src/transformState.js

```diff
@@ -46,7 +46,39 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  const copyState = state;
+
+  for (const key in transforms) {
+    const { operation, properties } = transforms[key];
+
+    if (operation === 'addProperties') {
```

Comment:
строки прийнято виносити в окрему змінну константу, для подальшого використання в кількох місцях.
 ```
const OPERATIONS = {
  add: 'addProperties',
  remove: 'removeProperties',
  clear: 'clear'
}
```
потім ти можеш використовувати це ось так 
```suggestion
    if (operation === OPERATIONS.add) {
```
І якщо ці операції будуть використовувати в 2-3 місцях в програмі і в разі якщо треба буде змінити строку на щось інше, тобі не треба буде шукати і робити зміни по всіх файлах, достатньо змінити тільки строку в константі.